I was unable to upload the data set onto the jadbio today(09-10-2023),but yesterday i was able to upload instead of it, i used DataRobot tool which is automl tool and created an app using that please refer this link: https://app.datarobot.com/applications/64fe9cfa2dd0520a3f2ca5eb/?token=GV5Xxc7bCju9B1jNj42ZFk-WYr_wyPPf3cHBOU8uAYY
